gui/default/syncthing/core/syncthingController.js

```diff
@@ -1768,6 +1769,7 @@ angular.module('syncthing.core')
                 })
                 .error(function (err) {
                     $('#folder-ignores textarea').val($translate.instant("Failed to load ignore patterns."));
+                    $('#folder-ignores textarea').removeAttr('disabled');
```

Comment:
Probably should not make this change. It's part of the other issue (editing ignore patterns that fail to load...)

Reply:
We've also already worked on the editing ignore patterns that fail to load

Reply:
I know, but lets make that a separate change & PR

Reply:
On the same branch or on another ?

Reply:
Separate issues => separate branches, separate PRs